src/components/message-list/_macro.spec.js

```diff
@@ -39,7 +41,6 @@ const EXAMPLE_MESSAGE_LIST = {
     ...EXAMPLE_MESSAGE_LIST_MINIMAL,
     ariaLabel: 'Message list for ONS Business Surveys',
     ariaLabelMetaData: 'Message information',
-    ariaLabelMsg: 'Message preview',
```

Comment:
As we still support this param we still need to test it. We will need to add an example in where this is set instead of `bodyLabel` and a test similar to the new test you have added